src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -101,6 +101,22 @@ export const BaseComment = ({
     const [isEditing, setIsEditing] = React.useState<boolean>(false);
     const [isInputOpen, setIsInputOpen] = React.useState<boolean>(false);
 
+    const prevStatus = React.useRef<?FeedItemStatus>(status);
```

Comment:
Do we need the ref here? Not sure if the previous status matters or not, we just need to make sure that the comment is in the resolved state if we want to call onHideReplies and replies and repliesTotalCount shouldn't change if the Comment is resolved.

Reply:
The issue is we still want to allow the user to expand or hide the replies manually even if its in the resolved state. If we remove the ref then the that functionality won't work anymore since the comments will always be forced to hide when the comment is resolved

Reply:
Ahh ok that's right. I re-read the AC. Looks good

Reply:
I was forced to add those other dependencies to `useEffect` by eslint. Initially I just wanted `status` as the dependency.